### PR TITLE
fix: log level

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

We noticed that no `session_replay` row in the `log_entries` table in ClickHouse had the associated `level`

## Changes

We were incorrectly producing Kafka messages with the key `log_level` but the column in the ClickHouse table is called `level`

## How did you test this code?

Updated tests

running locally and seeing it work

### CODE REVIEW

File: plugin-server/src/main/ingestion-queues/session-recording/process-event.ts [hunk 1]
```diff
@@ -145,6 +145,9 @@ export const gatherConsoleLogEvents = (
         }
     })
 
+    console.log('DAVID WAS ERE')
+    console.log(consoleLogEntries)
+
```
Comment (pauldambra) [inline plugin-server/src/main/ingestion-queues/session-recording/process-event.ts:150]:
```suggestion
```

🤣

**Review by pauldambra — APPROVED** [review]
LGTM

File: plugin-server/src/main/ingestion-queues/session-recording/process-event.ts (hunk 1 above)
Comment (daibhin) [reply plugin-server/src/main/ingestion-queues/session-recording/process-event.ts:150]: Cringe 😵

**Comment by pauldambra** [comment]
validated this locally 💖 

made me realise that we have a way too simple set of filters for the data we might be gathering... but that can be a follow up :)

**Comment by pauldambra** [comment]
@daibhin seeing the `safeString` method reminded me we'd had an incident because we managed to pass some values into the kafka table for CH that acted as a poison pill and blocked ingestion.

We've had similar for the session_replay_events table

(and I believe before my time the events table too)

So it made me ask myself - how much do I trust that we're receiving values that are safe... and I thought actually not at all 🤣

**Comment by daibhin** [comment]
@pauldambra looks good! Will merge in the morning so we can be around if anything goes awry

File: plugin-server/src/main/ingestion-queues/session-recording/process-event.ts [hunk 2]
```diff
@@ -45,10 +45,64 @@ export interface SummarizedSessionRecordingEvent {
     snapshot_source: string | null
 }
 
+// this is of course way more complicated than you'd expect
```
Comment (marandaneto) [inline plugin-server/src/main/ingestion-queues/session-recording/process-event.ts:48]: consider the mobile ones as well https://github.com/PostHog/posthog-android/blob/d57bc993755327482163d8fc1f7e2aa606c70cdd/posthog-android/src/main/java/com/posthog/android/replay/internal/LogcatParser.java#L56-L68

**Review by marandaneto — APPROVED** [review]

File: plugin-server/src/main/ingestion-queues/session-recording/process-event.ts (hunk 2 above)
Comment (pauldambra) [reply plugin-server/src/main/ingestion-queues/session-recording/process-event.ts:48]: it's going to take so long to get used to not only considering browser APIs 🤣 

luckily they overlap with the browser ones - phew!

**Pull request merged**